index.html

```diff
@@ -220,6 +220,66 @@ <h3>Multi-Post Stories</h3>
             </div>
           </section>
 
+          <!--Contact-form-->
+          <section id="contact" class="c-form">
+            <p class="con-us">CONTACT US</p>
+            <p class="con-par">
+              If you have an application you are
+              interested in developing, a feature that you
+              need built or a project that needs coding.
+              I'd love to help with it.
+            </p>
```

Comment:
- [ ] Make sure the **"If you have an appli..."** paragraph match the styling like on the Figma design such as its **color**, **font-weight**, **font-family**, **line-heigt**,.... Make sure you check all the properties on the Figma template to match your design to it.